packages/core/src/components/EmptyState/EmptyStateImage.tsx

```diff
@@ -54,6 +54,7 @@ export const EmptyStateImage = ({ missing }: Props) => {
           src={noInformation}
           alt="no Information"
           className={classes.generalImg}
+          data-testid="noInformation"
```

Comment:
Afaik it should be possible to select these via the alt text instead of adding a `testid`, using e.g. `getByAltText`. We tend to avoid adding `testid`s when possible :grin: